src/SourceGenerators/Uno.UI.SourceGenerators/Helpers/RoslynMetadataHelper.cs

```diff
@@ -195,11 +195,6 @@ private ITypeSymbol SourceFindTypeByFullName(string fullName)
 				}
 			}
 
-			if (symbol?.Kind == SymbolKind.ErrorType)
-			{
-				return null;
-			}
-
```

Comment:
Why do you need to remove this part?

Reply:
This condition can never be true. It's duplicate from the one in line 149, and `symbol` was never re-assigned in between.

The change isn't necessary for performance, but I included it as a small cleanup